### PR TITLE
Removed horizontal scroll on individual course pages

### LINKED ISSUE #281: UX Bug — Course Page has horizontal scrollbar on browser
**Describe the bug**
The CoursePage has a horizontal scrollbar for extra right margin in browser
Screen Shot 2020-10-31 at 1 19 55 AM

**To Reproduce**
1. Go to explore
2. Click on a department
3. Click on a course

**Expected behavior**
Should be no extra space on the right and no horizontal scrollbar

### PR DESCRIPTION
**Pull request by nick-lennox**
closes #281 , simple fix to remove the horizontal scroll on the individual course page by adding 0px margin to b-rows.

### CODE REVIEW

**Review by jshom — APPROVED** [review]
lgtm! nice refactor

**Comment by jshom** [comment]
@YuetingLiao @IkeLyons this is probs also a solution to the standard padding issue we were having earlier, appears the other pages also don't have `<b-container>` elements

File: src/web/src/pages/CoursePage.vue [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
-        {{ courseObj.description }}
-      </b-col>
-    </b-row>
-    <b-button :to="backRoute">Back</b-button>
-  </div>
-  <div v-else>
-    <b-spinner></b-spinner>
-    <strong class="m-2">Loading Course...</strong>
-  </div>
+  <b-container fluid>
```
Comment (YuetingLiao) [inline src/web/src/pages/CoursePage.vue:2]: Nice, does the horizontal scroller disappears after adding the ` b-container`?

**Comment by YuetingLiao** [comment]
> @YuetingLiao @IkeLyons this is probs also a solution to the standard padding issue we were having earlier, appears the other pages also don't/didn't have `<b-container>` element

Can you remind me about what was the standard padding issue? Thx.

**Review by YuetingLiao — APPROVED** [review]

**Pull request merged**